packages/@office-iss/react-native-win32/src-win/Libraries/Components/TextInput/TextInput.win32.js

```diff
@@ -1917,20 +1928,6 @@ const ExportedForwardRef: React.AbstractComponent<
   },
   forwardedRef: ReactRefSetter<TextInputInstance>,
 ) {
-  // $FlowFixMe[underconstrained-implicit-instantiation]
```

Comment:
Was this style change a windows only thing or was this from upstream?

Reply:
Upstream!